frontend/src/shared/helpers.tsx

```diff
@@ -990,6 +995,9 @@ export function prepareSampleCohortDataForAgGrid(samples: Sample[]) {
       });
     });
     const deliveryDate = cohortDates?.sort()[0]; // earliest cohort date
+    var embargoDateAsDate = new Date(deliveryDate);
```

Comment:
@ao508 i like to use a library called Moment.js https://momentjs.com/ for this kind of date handling stuff.  But up to you.